app/main/posts/modify/post-editor.directive.js

```diff
@@ -139,6 +134,24 @@ function PostEditorController(
                     }
                     $scope.medias[attr.key] = {};
                 }
+                // assingning tag-objects to attribute-options
+                if (attr.input === 'tags') {
```

Comment:
May be we could chat through this structure, not positive I follow it, but can comment better when we take a quick look together